.travis.yml

```diff
@@ -30,6 +30,7 @@ matrix:
       python: '3.7'
       install:
         - pip install -U pip setuptools
+        - pip install -U importlib-metadata~=5.0 # This must be installed before parsing the setup.py file
```

Comment:
Can we remove this to avoid confusion. If this is now pinned in setup.py it shouldn't be needed.

Reply:
The pipeline will still fail if we remove the line in the travis file as well. It is needed there because to even start parsing the setup.py file the proper version of importlib-metadata is needed